.github/workflows/e2e.yml

```diff
@@ -51,6 +52,9 @@ jobs:
         docker-compose -f docker-compose.yml -f expose-cassandra-port.yml up -d scalar-ledger ledger-envoy
         sleep 5
 
+    - name: Update NPM to the latest stable version
+      run: npm install -g npm@latest
+
```

Comment:
This step installs the latest stable NPM (which is version 8 now).
The reason for doing this is the default NPM CLI of Node.js version 14 is version 6 which doesn't support package-lock.json version 2. So, I think always using the latest stable NPM CLI might be a good idea.